x-pack/plugins/enterprise_search/public/applications/workplace_search/index.tsx

```diff
@@ -95,6 +96,9 @@ export const WorkplaceSearchConfigured: React.FC<InitialAppData> = (props) => {
           <Route path={PERSONAL_SETTINGS_PATH}>
             <AccountSettings />
           </Route>
+          <Route>
+            <NotFound isOrganization={false} />
+          </Route>
```

Comment:
WS personal dashboard top level 404

<img width="1325" alt="" src="https://user-images.githubusercontent.com/549407/123198248-9e702d00-d461-11eb-8710-e73474552e82.png">

This is pretty fugly, I know... Sorry, I don't have bandwidth right now to mess with it and it's better than before when it was just a blank page, right? 🤪

Reply:
New personal dashboard 404 now with the `enterpriseSearchLogo`:

<img width="1322" alt="" src="https://user-images.githubusercontent.com/549407/123298438-5b4ba380-d4cd-11eb-915f-f7cdaa3697bf.png">

We probably should take a look at this after FF, haha 🙈